e2e/utils/waits-utils.ts

```diff
@@ -44,10 +44,11 @@ export async function waitForDocumentTitle(page: Page, titleSubstr: string): Pro
   try {
     const jsHandle = await page.waitForFunction(
       (t) => {
+        const regExp = new RegExp(t);
         const actualTitle = document.title;
-        return actualTitle.includes(t);
+        return actualTitle && regExp.test(actualTitle);
       },
-      { timeout: 10 * 60 * 1000 },
+      { timeout: 60 * 1000 },
```

Comment:
why reduce this timeout?

Reply:
I think a long timeout is hiding problem if a page really takes that long to load. IMO, no page should take 10 min to load. Also, long timeout makes longer to fail if fails here.